libraries/AP_OSD/AP_OSD.cpp

```diff
@@ -278,3 +281,21 @@ void AP_OSD::next_screen()
     } while (t != current_screen && !screen[t].enabled);
     current_screen = t;
 }
+
+void AP_OSD::stats()
```

Comment:
Better use existing code to deal with is_flying detection.

Reply:
dont know how...need help on that